packages/web-pkg/src/composables/keyboardActions/useInterceptModifierClick.ts

```diff
@@ -0,0 +1,31 @@
+import { eventBus } from '../../services'
+import type { Resource } from '@opencloud-eu/web-client'
+
+export function useInterceptModifierClick(event: MouseEvent, resource: Resource): boolean {
```

Comment:
Generally, composables are more used a wrappers for functionality. So in this case, `useInterceptModifierClick` would expose a method (e.g. `interceptModifierClick`) that then takes these arguments and handles the logic.

You then use this composable in setup functions or inside other composables like this: `const { interceptModifierClick } = useInterceptModifierClick()`. Not directly inside methods, templates or computed properties, but in the root of a setup function or another composable. The method itself gets then called where you need to call it.

The same goes for the `useToggleTile` composable.